packages/@aws-cdk/aws-batch-alpha/lib/ecs-container-definition.ts

```diff
@@ -618,6 +638,10 @@ abstract class EcsContainerDefinitionBase extends Construct implements IEcsConta
     this.user = props.user;
     this.volumes = props.volumes ?? [];
 
+    if (props.runtimePlatform) {
+      this.runtimePlatform = props.runtimePlatform;
+    }
+
```

Comment:
I think you should modify the  _renderContainerDefinition() function to add the new `runtimePlatform` property like

```
runtimePlatform: this.runtimePlatform
```

So it will synth a new container definition for [ContainerProperties](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-batch-jobdefinition-containerproperties.html)